custom_connectors/oauth2/ephesoft_connector.rb

```diff
@@ -0,0 +1,225 @@
+          }
+        )
+      end,
+
+      webhook_notification: lambda do |input, payload|
+        payload
+      end,
+
+      webhook_unsubscribe: lambda do |webhook|
+
+        # Get the webhook ID to be removed
+        json_output = webhook['data']
+        configurationId = (json_output.scan(/configurationId"=>"([\S\s]*)", "integrationId/).dig(0))[0]
+
+        # Delete the webhook
+        delete("https://api.us.ephesoft.io/v1/settings/integrations/configurations/#{configurationId}")
+      end,
+
+      output_fields: lambda do |object_definitions|
+        object_definitions["webhook_output"]
+      end,
+      sample_output: lambda do ||
+        {
+          AmountDue: '3693.80',
+          CustomerId: '123',
+          DocumentId: '00000000-0000-0000-0000-000000000000',
+          DueDate: '2020-04-15',
+          InvoiceDate: '2020-03-29',
+          InvoiceNumber: '9291873611',
+          OrderDate: '2020-03-29',
+          PdfUrl: 'https://semantik.us.ephesoft.io/assets/samples/webhookTestInvoice.pdf',
+          PONumber: '8762943181',
+          ShipDate: '2020-04-10',
+          ShipFreight: '123.45',
+          SubTotal: '3358',
+          TableUrl: 'https://semantik.us.ephesoft.io/assets/samples/webhookTestInvoiceTable.json',
+          TaxAmount: '335.80',
+          TaxId: '012-34-56789',
+          TaxRate: '10',
+          TenantId: '555e4d7e-0115-4ec4-adac-892367844222',
+          Terms: 'Net 30 days',
+          TotalAmount: '3693.8',
+          Vendor: {
+            VendorAddress: {
+              VendorStreetAddress: '2775 S. 900 W.',
+              VendorPOBox: '123',
+              VendorLocality: 'Salt Lake City',
+              VendorRegion: 'UT',
+              VendorPostalCode: '84128-3227',
+              VendorCountry: 'US'
+            },
+            VendorCustom1: 'Custom1',
+            VendorCustom2: 'Custom2',
+            VendorCustom3: 'Custom3',
+            VendorCustom4: 'Custom4',
+            VendorCustom5: 'Custom5',
+            VendorIBAN: 'aa5d5efa-fcaf-4d2b-a118-a093f184a876',
+            VendorCustomerID: '95bba4ab-1897-4e9d-aff1-f42ae7e5cb86',
+            VendorID: '1',
+            VendorMatched: 'true',
+            VendorName: 'Kord Industries',
+            VendorStatus: 'active',
+            VendorSWIFT: '1b0f74e1-ce62-4d1d-9153-f07e88bc0a29',
+            VendorTelephone: '513-101-4074'
+          }
+        }
+      end,
+      dedup: lambda do |messages|
+        Time.now.to_f
+      end
+    }
+  },
+  object_definitions: {
+    webhook_output: {
+      fields: lambda do
+        [
+          { name: "AmountDue"},
```

Comment:
Output definitions are missing the `type` declarations. 

https://docs.workato.com/developing-connectors/sdk/object-definition.html#structuring-input-and-output-fields

Reply:
Added types in latest commit.  As an FYI: I was previously following the sample code on the referenced page which only adds the type declarations to non-string types.  All of these are strings which is why I didn't add any earlier.